xapian-letor/api/featurelist.cc

```diff
@@ -41,13 +43,21 @@ FeatureList::FeatureList() : internal(new FeatureList::Internal())
     internal->feature.push_back(new CollTfCollLenFeature());
     internal->feature.push_back(new TfIdfDoclenFeature());
     internal->feature.push_back(new TfDoclenCollTfCollLenFeature());
+    for (Feature* it : internal->feature) {
+	internal->stats_needed = Internal::stat_flags(internal->stats_needed
+					    | it->get_stats());
```

Comment:
The Xapian convention for breaking a line at most binary operators (such as `|` here) is that the operator goes before the break, and the start of the two arguments of the operator are then vertically aligned to make it easier for the reader to see what they are (so the "i" of  `it->...` should line up under the "i" of `internal->...`).

(iostream `<<` is treated differently - while that's actually a binary operator, it really reads more like `<<` applied to the thing to its right).

Reply:
fixed in a fixup commit